ts/src/kucoin.ts

```diff
@@ -3798,7 +3883,7 @@ export default class kucoin extends Exchange {
         const version = this.safeString (params, 'version', defaultVersion);
         params = this.omit (params, 'version');
         let endpoint = '/api/' + version + '/' + this.implodeParams (path, params);
-        if (api === 'webFront') {
+        if (api === 'webExchange') {
```

Comment:
`webFront` was only mentioned in  kucoinfutures, and I've replaced it in all mentions